runtime/runc_jailer.go

```diff
@@ -246,15 +246,30 @@ func (j *runcJailer) BuildLinkFifoHandler() firecracker.Handler {
 			contentsPath := j.RootPath()
 			fifoFileName := filepath.Base(m.Cfg.LogFifo)
 			newFifoPath := filepath.Join(contentsPath, fifoFileName)
-			if err := os.Link(m.Cfg.LogFifo, newFifoPath); err != nil {
-				return err
+
+			fifoFd, err := os.Create(newFifoPath)
+			if err != nil {
+				return errors.Wrapf(err, "failed to create %s", newFifoPath)
+			}
+			if err := fifoFd.Close(); err != nil {
+				j.logger.WithError(err).Debug("failed to close fifo file")
```

Comment:
How about returning the error instead of just logging? I'm unsure what would cause the error.